### PR TITLE
Several fixes in CI/CD

### PR DESCRIPTION
**Pull request by Jesus89**
# Description

Shortcut

- Story: https://app.shortcut.com/cartoteam/story/266416

## Type of change

- Fix

### CODE REVIEW

File: .github/workflows/databricks.yml [hunk 1]
```diff
@@ -27,17 +34,13 @@ jobs:
...
           credentials_json: ${{ secrets.CARTODB_ON_GCP_TERRAFORM_CI_CD_SERVICE_ACCOUNT_KEY }}
-
       - id: 'secrets'
         uses: 'google-github-actions/get-secretmanager-secrets@v0'
         with:
           secrets: |-
             databricks-ci-token:projects/cartodb-on-gcp-terraform-ci-cd/secrets/terraform-databricks-aws-ciuser-token
       - name: Check diff
         uses: technote-space/get-diff-action@v4
-        with:
```
Comment (jgoizueta) [inline .github/workflows/databricks.yml:38]: why is this removed?

**Review by jgoizueta — APPROVED** [review]

File: .github/workflows/databricks.yml (hunk 1 above)
Comment (Jesus89) [reply .github/workflows/databricks.yml:38]: We are not using the filter of get-diff in the other clouds. Just a clean up to align scripts

**Review by aarroyosal — APPROVED** [review]
